src/openfermion/measurements/_hamiltonian_stabilizers_sets.py

```diff
@@ -0,0 +1,134 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Tools to split the Hamiltonian into subsets based on the support of
+its stabilizers."""
+
+import numpy
+import copy
+from openfermion.ops import QubitOperator
+
+
+def _check_stabilizer_overlap(pauli_op, stabilizer):
```

Comment:
This function doesn't really have anything to do with stabilizers. So rename it to `_has_overlapping_indices(op1, op2)`, and remove references to stabilizers in the docstring.

Reply:
Indeed, the functions only aims to check if two Pauli operators have common qubits.

Updated name and docstring.